client/src/app/pages/applications/applications-table-analyze/applications-table-analyze.tsx

```diff
@@ -570,12 +570,15 @@ export const ApplicationsTableAnalyze: React.FC = () => {
           <Thead>
             <Tr>
               <TableHeaderContentWithControls {...tableControls}>
-                <Th {...getThProps({ columnKey: "name" })} />
-                <Th {...getThProps({ columnKey: "description" })} />
-                <Th {...getThProps({ columnKey: "businessService" })} />
-                <Th {...getThProps({ columnKey: "analysis" })} />
-                <Th {...getThProps({ columnKey: "tags" })} />
-                <Th {...getThProps({ columnKey: "effort" })} />
+                <Th {...getThProps({ columnKey: "name" })} width={20} />
+                <Th {...getThProps({ columnKey: "description" })} width={25} />
+                <Th
+                  {...getThProps({ columnKey: "businessService" })}
+                  width={10}
+                />
+                <Th {...getThProps({ columnKey: "analysis" })} width={10} />
+                <Th {...getThProps({ columnKey: "tags" })} width={10} />
+                <Th {...getThProps({ columnKey: "effort" })} width={10} />
                 <Th />
```

Comment:
mentioned in slack but these numbers appear to be % that should add up to 100, also we should probably add to actions column as well